test/FunctionalTests/PublishForRidTest_CoreCLR.cs

```diff
@@ -1,72 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.IO;
-using System.Runtime.InteropServices;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Server.IntegrationTesting;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Logging.Testing;
-using Xunit;
-using Xunit.Abstractions;
-
-namespace FunctionalTests
-{
-    public class PublishWithRidTest_CoreCLR : LoggedTest
-    {
-        public PublishWithRidTest_CoreCLR(ITestOutputHelper output)
-            : base(output)
-        {
-        }
-
-        [Fact]
-        public void CrossPublishingWorks()
```

Comment:
? killed the whole test?

Reply:
We discussed this - the test is some what marginal value and sounds like making it work would be painful.